### ISSUE TITLE
Allow community to vote if package works on Umbraco Cloud

### ISSUE THREAD

**Issue by bjarnef** [umbraco/OurUmbraco#626]
I just had a look at the Umbraco Cloud category on Our, but it only seems a handful packages supports Umbraco Cloud (or has been tested works on Umbraco Cloud): https://our.umbraco.com/packages/?category=uaas

However I think this a bit misleading since some may think that there are not many packages supported on Umbraco Cloud (and maybe a reason for not selecting Umbraco Cloud). Furthermore only Umbraco HQ can mark packages as "Works on Umbraco Cloud", but the community probably know better which packages that works on Umbraco Cloud.

As far as I know the list of packages marked as  "Works on Umbraco Cloud" hasn't been updated for a while. E.g. Vendr works very well on Umbraco Cloud although it doesn't support Deploy yet: https://our.umbraco.com/packages/website-utilities/vendr/

Maybe there should be a "Works on Umbraco Cloud" with a separate vote/score next to it? Maybe using green/orange/red depending on the percentage of the community who has reported it works on Umbraco Cloud.

There could be a star or something similar next to it to indicate that Umbraco HQ has confirmed this?

**Comment by AaronSadlerUK** [comment]
I also think the Author should be able to mark this.

For example I know MegaNavV8 works on Umbraco cloud, including with content migrations.

Being able to mark this as tested on Umbraco Cloud would be a bonus

**Comment by nul800sebastiaan** [comment]
Hi @bjarnef and @AaronSadlerUK - we will happily mark things as compatible with Cloud as soon as we've made sure that claim is correct. We want to keep some quality control on that for the moment since we'll be getting the complaints if it doesn't work, our procedure is currently as documented here: https://our.umbraco.com/documentation/Umbraco-Cloud/Frequently-Asked-Questions/#do-you-support-package-x-on-umbraco-cloud

That said, a voting mechanism is not a bad idea to add extra trust to that claim, but we'll need to figure out at HQ how this should work. At the moment that's not on our roadmap. I'll mark this as an idea we'll have a look at in the future.

Thanks for the feedback so far and of course any other related ideas are more than welcome in the comments here.

**Comment by bjarnef** [comment]
@nul800sebastiaan sure, but then there has also been packages like Multi Url Picker which was marked as "Works on Umbraco Cloud", but no longer was supported on Umbraco Cloud when the switch from Courier to Deploy was made https://our.umbraco.com/packages/backoffice-extensions/multi-url-picker/
This mark was removed after reaching out to Umbraco Support Team.. but later updated to support Deploy... and later again introduced in core.

Looking at the list of packages "Works in Umbraco Cloud" are only a few of packages which actually do work on Umbraco Cloud and I fear this for some could be a reason not selecting Umbraco Cloud.

Maybe the community could vote if a package works on Umbraco Cloud and instead have a badge/icon or something to visualize packages which has been tested and confirmed works on Umbraco Cloud?

**Comment by nul800sebastiaan** [comment]
Yep all good remarks. It doesn't fit right now but we'll look at it in the future!

**Comment by dawoe** [comment]
@nul800sebastiaan how can a package dev get HQ to test to check if the package works on Cloud ?

Because in the link you posted describing the procedure, it seems you need to reach out to support using the cloud portal. Not all package developers have access to that.

I only have one of my packages listed as working for Cloud. But I know from experience all of them work on Cloud. I actually don't know how my OEmbed editor ever got that seal of approval ?

**Comment by LottePitcher** [comment]
There is an open issue - https://github.com/umbraco/Umbraco.Packages/issues/53 - suggesting that the ability to 'request a Cloud review' is added to the packages area on Our. This should make it easier for package devs to start the review process.
